packages/grid_client/src/storage/tfkvstore.ts

```diff
@@ -15,16 +15,33 @@ class TFKVStoreBackend implements BackendStorageInterface {
     this.client = new TFClient(url, mnemonic, storeSecret, keypairType);
   }
 
+  private async cleanFragments(key: string, start: number) {
+    let k = `${key}.${start}`;
+    let value = await this.client.kvStore.get({ key: k });
+    if (!value) {
+      return [];
+    }
+    const extrinsics: SubmittableExtrinsic<"promise", ISubmittableResult>[] = [];
+    while (value) {
+      extrinsics.push(await this.client.kvStore.delete({ key: k }));
+      start++;
+      k = `${key}.${start}`;
```

Comment:
Do we need to reassign the `k` variable? I see that you have already incremented the 'start' value +=1.

Reply:
Nevermind, I realized that I was mistaken and confused about the concept of pure JavaScript and VueJS's ref concept xD